install/monitoring.md

```diff
@@ -3,7 +3,9 @@ Monitoring
 
 The Percona Monitoring and Management (PMM) [provides an excellent solution](https://www.percona.com/doc/percona-xtradb-cluster/LATEST/manual/monitoring.html#using-pmm) to monitor Percona XtraDB Cluster.
 
-Following steps are needed to install both PMM Client and PMM Server preconfigured to monitor Percona XtraDB Cluster on Kubernetes or OpenShift.
+### Installing the PMM Server
+
+Following steps are needed to install the PMM Server to monitor Percona XtraDB Cluster on Kubernetes or OpenShift.
```

Comment:
It is needed to mention that -- This step is optional If you already have PMM Server installed (inside or outside of Kubernetes)